package.json

```diff
@@ -42,6 +42,7 @@
         "protractor": "~5.4.0",
         "ts-node": "~7.0.0",
         "tslint": "~5.15.0",
-        "typescript": "~3.5.3"
+        "typescript": "~3.5.3",
+        "tslib": "^1.10.0"
```

Comment:
Why did you add this dependency? Did you have any problems compiling the app? Any TypeScript Compiler related error? If not,  this is out of the ticket scope.

Reply:
I had some problems using the linter and it was recommended to add that dependency.